### PR TITLE
adding AzurePowershellV5

### PR DESCRIPTION
**Pull request by 20shivangi**

### CODE REVIEW

File: Tasks/Common/VstsAzureHelpers_/Utility.ps1 [hunk 1]
```diff
@@ -12,7 +12,7 @@
         $pemFileContent = $Endpoint.Auth.Parameters.ServicePrincipalCertificate
         $pfxFilePath, $pfxFilePassword = ConvertTo-Pfx -pemFileContent $pemFileContent
         
-        $certificate.Import($pfxFilePath, $pfxFilePassword, [System.Security.Cryptography.X509Certificates.X509KeyStorageFlags]::PersistKeySet)
```
[the remark this answers is not in the document]
Comment (20shivangi) [reply Tasks/Common/VstsAzureHelpers_/Utility.ps1]: It is done

File: Tasks/AzurePowerShellV5/AzurePowerShell.ps1 [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+{
+    # Generate the script contents.
+    Write-Host (Get-VstsLocString -Key 'GeneratingScript')
+    $contents = @()
+    $contents += "`$ErrorActionPreference = '$__vsts_input_errorActionPreference'"
+    if ($env:system_debug -eq "true") {
+        $contents += "`$VerbosePreference = 'continue'"
+    }
+
+    $CoreAzArgument = $null;
```
[the remark this answers is not in the document]
Comment (20shivangi) [reply Tasks/AzurePowerShellV5/AzurePowerShell.ps1]: As it calls CoreAz.ps file, so while passing argument to that file, I named the arguments as coreAzArgument

File: Tasks/AzurePowerShellV5/task.loc.json [hunk 3]
```diff
@@ -0,0 +1,190 @@
...
+    "Release"
+  ],
+  "runsOn": [
+    "Agent",
+    "DeploymentGroup"
+  ],
+  "author": "Microsoft Corporation",
+  "version": {
+    "Major": 5,
+    "Minor": 162,
```
Comment (niadak) [inline Tasks/AzurePowerShellV5/task.loc.json]: make it preview

File: Tasks/AzurePowerShellV5/AzurePowerShell.ps1 (hunk 2 above)
[the remark this answers is not in the document]
Comment (20shivangi) [reply Tasks/AzurePowerShellV5/AzurePowerShell.ps1]: Resolved offline

File: Tasks/AzurePowerShellV5/task.loc.json (hunk 3 above)
Comment (20shivangi) [reply Tasks/AzurePowerShellV5/task.loc.json]: It is done

**Comment by eaarora-ms** [comment]
Fixes : https://github.com/microsoft/azure-pipelines-tasks/issues/11672

**Pull request merged**